tests/test_solvermode/test_initialization.py

```diff
@@ -5,7 +5,7 @@
 @pytest.mark.nightly
 @pytest.mark.quick
 @pytest.mark.setup
-@pytest.mark.fluent_version("latest")
+@pytest.mark.fluent_version("==23.2")
```

Comment:
This test compares settings state dict and is too sensitive to API change. I don't think we need to test it with the dev version.